projects/canopy/src/styles/grid.scss

```diff
@@ -49,6 +49,53 @@ $columns: 12;
   max-width: 100%;
 }
 
+@mixin lg-grid-alignment($breakpoint) {
```

Comment:
I would probably create directives for these too

Reply:
See https://github.com/Legal-and-General/canopy/issues/441